integration-test/2020-locality-osm-ne-join.py

```diff
@@ -0,0 +1,161 @@
+# -*- encoding: utf-8 -*-
+import dsl
+
+from . import FixtureTest
+
+
+class OSMNEJoinTest(FixtureTest):
+    def test_ne_pop_max_override(self):
+        z, x, y = (16, 10482, 25330)
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/node/26819236
+            dsl.point(26819236, (-122.4199061, 37.7790262), {
+                'name': u'San Francisco',
+                'place': u'city',
+                'population': u'864816',
+                'rank': u'10',
+                'short_name': u'SF',
+                'source': u'openstreetmap.org',
+                'wikidata': u'Q62',
+                'wikipedia': u'en:San Francisco',
+                '__ne_pop_max': u'20000000'
```

Comment:
Let's use real values:

> 3450000

Reply:
To figure out what values to use... you need to download linked file from the web, and view it in QGIS, from:

https://www.naturalearthdata.com/downloads/10m-cultural-vectors/10m-populated-places/

And query this city.